packages/appstore/src/modules/wallets/wallets.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { Loading, ThemedScrollbars } from '@deriv/components';
+import { useActiveWallet, useWalletsList } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import AddMoreWallets from 'Components/add-more-wallets';
+import ModalManager from 'Components/modals/modal-manager';
+import DesktopWalletsList from './desktop-wallets-list';
+import MobileWalletsCarousel from './mobile-wallets-carousel';
+import './wallets.scss';
+
+const Wallets = observer(() => {
+    const { client, ui } = useStore();
+    const { switchAccount, is_authorize } = client;
```

Comment:
Looks like `is_authorize` is unused.
```suggestion
    const { switchAccount } = client;
```